### PR TITLE
Enable packer-build to replicate AMIs and share them

### PR DESCRIPTION
**Pull request by gene1wood**

### CODE REVIEW

File: cloudy_mozdef/Makefile [hunk 1]
```diff
@@ -48,7 +49,7 @@ create-dev-s3-bucket:
 	@export AWS_REGION=$(AWS_REGION)
 	aws s3api create-bucket --bucket $(S3_DEV_BUCKET_NAME) --acl public-read --create-bucket-configuration LocationConstraint=$(AWS_REGION)
 
-.PHONY: updated-dev-stack
+.PHONY: update-dev-stack
```
Comment (andrewkrug-mozilla-owner) [inline cloudy_mozdef/Makefile]: Fixed typo :p

File: cloudy_mozdef/Makefile [hunk 2]
```diff
@@ -48,7 +49,7 @@ create-dev-s3-bucket:
 	@export AWS_REGION=$(AWS_REGION)
 	aws s3api create-bucket --bucket $(S3_DEV_BUCKET_NAME) --acl public-read --create-bucket-configuration LocationConstraint=$(AWS_REGION)
 
-.PHONY: updated-dev-stack
```
Comment (andrewkrug) [inline cloudy_mozdef/Makefile]: Fixed typo :p

File: cloudy_mozdef/Makefile [hunk 3]
```diff
@@ -27,10 +27,11 @@ all:
 	@echo 'Run ./dmake <target> in order to run the Makefile targets in Docker'
 
 # Note: This requires AWS access
+# https://blog.gruntwork.io/locating-aws-ami-owner-id-and-image-name-for-packer-builds-7616fe46b49a
 .PHONY: packer-build
 packer-build: ## Build the base AMI with packer
 	@echo "Branch based build triggered for $(BRANCH)..."
-	cd packer && packer -machine-readable build -var github_branch=${BRANCH} packer.json
+	ci/pack_and_copy ${BRANCH}
```
Comment (andrewkrug) [inline cloudy_mozdef/Makefile]: Can you add a message that makes this reflect what we talked about yesterday?  It should notify the caller that this is being built using source "from github" and not local to set expectations.

Comment (gene1wood) [reply cloudy_mozdef/Makefile]: Good call, fixed in b85ed2a

**Pull request merged**